### PR TITLE
Add vertical swap icon

### PR DESCRIPTION
**Pull request by georgegillams**
Depends on https://github.com/Skyscanner/backpack/pull/1573

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Unreleased
 
+**Breaking**:
+  - `BpkIcon`:
+    - Renamed `bpk_swap` to `bpk_swap__horizontal`:
```
Comment (tiagoengel) [inline UNRELEASED.md]: It seems like the `bpk_swap` icon was not removed, can you try running `npm run build`

Comment (georgegillams) [reply UNRELEASED.md]: `npm run build` didn't remove anything. Where are you seeing it persisting?

Comment (tiagoengel) [reply UNRELEASED.md]: if `bpk-swap` was removed then you need to remove `Backpack/src/main/res/drawable-nodpi/bpk_swap.xml` too. I guess this is a limitation of our script.

Comment (georgegillams) [reply UNRELEASED.md]: I already did tho
https://github.com/Skyscanner/backpack-android/pull/317/files#diff-77a61cb12bc52ad8ed6d931f331f9ecd

Comment (georgegillams) [reply UNRELEASED.md]: Githubs showing it as a rename instead of delete because it recognises the content is the same...

Comment (tiagoengel) [reply UNRELEASED.md]: I right, sorry I didn't see it

**Pull request merged**